nodejs/toolchain.bzl

```diff
@@ -89,6 +105,19 @@ node_toolchain = rule(
             doc = "Path to an existing nodejs executable for the target platform.",
             mandatory = False,
         ),
+        "npm": attr.label(
+            doc = "A hermetically downloaded npm executable target for the target platform.",
```

Comment:
nit: should these really say for the "target platform"?

Reply:
right, the node fields are `target_tool` and `target_tool_path`.

`target_npm`, `target_npm_path` and `target_npm_files`?

Reply:
no I think the fields are named fine, it's just the docstring that's a tad misleading (for node as well - it was an existing problem here)